pom.xml

```diff
@@ -69,27 +69,27 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-step-api</artifactId>
-            <version>2.7</version>
+            <version>2.13</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.8</version>
+            <version>2.23.1</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-support</artifactId>
-            <version>2.12</version>
+            <version>2.14</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-durable-task-step</artifactId>
-            <version>2.8</version>
+            <version>2.16</version>
```

Comment:
Not 2.17?

Reply:
Again, last release to not use core 2.60.x: https://github.com/jenkinsci/workflow-durable-task-step-plugin/commit/f4ed021d3689a1a23a26d1bcf94508f8f1bf9049